rerun_py/rerun_sdk/rerun/components/__init__.py

```diff
@@ -104,3 +100,13 @@ def build_dense_union(data_type: pa.DenseUnionType, discriminant: str, child: pa
 
     except ValueError as e:
         raise ValueError(e.args)
+
+
+# NOTE: This has to live here for now, while we migrate to archetypes (circular experimental imports).
+def splat() -> Any:
```

Comment:
would still appreciate calling this `instance_key_splat` to make it clear what we're splatting